rs/ic_os/utils/src/deployment.rs

```diff
@@ -106,22 +131,52 @@ mod test {
                 "url": "https://dfinity.org/"
               },
               "resources": {
-                "memory": "490"
+                "memory": "490",
+                "cpu": "kvm"
               }
             }
         )
     });
 
-    static DEPLOYMENT_STRUCT: Lazy<DeploymentJson> = Lazy::new(|| {
+    const DEPLOYMENT_STR_NO_CPU: &str = r#"{
+  "deployment": {
+    "name": "mainnet"
+  },
+  "logging": {
+    "hosts": "elasticsearch-node-0.mercury.dfinity.systems:443 elasticsearch-node-1.mercury.dfinity.systems:443 elasticsearch-node-2.mercury.dfinity.systems:443 elasticsearch-node-3.mercury.dfinity.systems:443"
```

Comment:
Could you change this too?

Reply:
All the X_STR strings in deployment.rs are variations of the deployment.json files that currently exist on mainnet, so we'll want to keep those all the same for the unit tests.